cv_checker_frontend/src/services/http.service.ts

```
@@ -0,0 +1,55 @@
+    // Make the API call using fetch
+    const response = await fetch(fullUrl, {
+      method: type,
+      headers,
+      body: type === 'GET' ? null : body,
+      credentials: 'include',
+    });
+
+    const result = await response.json();
+    return result;
+  } catch (err) {
+    throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)